libs/providers/flagd/src/e2e/step-definitions/flagSteps.ts

```diff
@@ -63,6 +64,10 @@ export const flagSteps: Steps =
       expect(state.details?.reason).toBe(expectedReason);
     });
 
+    then(/^the error-code should be "(.*)"$/, (expectedError: keyof typeof ErrorCode) => {
+      expect(state.details?.errorCode).toBe(ErrorCode[expectedError]);
+    });
+
```

Comment:
New binding for updated gherkin suite: https://github.com/open-feature/flagd-testbed/commit/d31f2f9179d8224ca606f233b6d3d918cf94c909